src/components/header/Header.js

```diff
@@ -1,15 +0,0 @@
-import React from 'react'
-import "./Header.css";
-import HeaderLeft from './HeaderLeft';
-import HeaderRight from './HeaderRight';
-
-const Header = ({ title, image, content }) => {
```

Comment:
nice but like why do you do this sometimes and not other times, consistency is key